pkg/csi_driver/controller_test.go

```diff
@@ -456,7 +459,48 @@ func TestGenerateNewFileInstance(t *testing.T) {
 				Location: "foo-location",
 				Tier:     "foo-tier",
 				Network: file.Network{
-					Name: "foo-network",
+					Name:        "foo-network",
+					ConnectMode: directPeering,
+				},
+				Volume: file.Volume{
+					Name:      newInstanceVolume,
+					SizeBytes: testBytes,
+				},
+			},
+		},
+		{
+			name: "custom params, private connect mode",
```

Comment:
can you also add invalid param test case for the invalid connectMode

Reply:
done